public/app.js

```diff
@@ -0,0 +1,192 @@
+var articlesArr = []
+var savedArticlesArr = []
+
+function findArticleById(artPubId, articlesArr) {
+  let tgtArticle = null
+  for (let i = 0; i < articlesArr.length; i++) {
+    let article = articlesArr[i]
+    if (article.pubId === artPubId) {
+      // found
+      tgtArticle = article
+      break
+    }
+  }
+  return tgtArticle
+}
+
+function btnScrapeClick() {
```

Comment:
theres a bug with your click events, I would take a look a the event listener functions again